jep/1/README.adoc

```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
 community and discuss important objections or concerns raised
 during discussion.
 
-. **Backwards Compatibility** -
-  All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the sponsor proposes to deal with these incompatibilities.
-  If there are no dackwards compatibility concerns, the section must say that.
-  JEP submissions that do not adequately discuss backwards compatibility,
-  when such discussion is required will not be approved as JEP Drafts.
+. **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
+  The JEP must explain how the it proposes to deal with these incompatibilities.
+  If there are no backwards compatibility concerns, the section must say that.
+. **Security** - All JEPs must include a section describing any security risks and their severity.
+  The JEP must explain how the it proposes to deal with these risks.
+  If there are no security concerns, the section must say that.
+. **Infrastructure Requirements** -
+  All JEPs must include a section describing any infrastructure changes or addtions they require.
```

Comment:
As the security section this is too narrow. E.g. (and IIRC) @rtyler was annoyed by nobody discussing with him impact of the setup wizard before we rolled out 2.0 and it hammered the update sites.

So, more generally, *impact on* Jenkins project infrastructure. (Adapt the next few lines as needed.)